packages/node/src/plugins/loadScript.ts

```diff
@@ -58,17 +46,32 @@ export const executeLoadTemplate = `
     if(!name) {
       throw new Error('__webpack_require__.l name is required for ' + url);
     }
+
     if (typeof global.__remote_scope__[name] !== 'undefined') return callback(global.__remote_scope__[name]);
+
     const vm = require('vm');
     (global.webpackChunkLoad || global.fetch || require("node-fetch"))(url).then(function (res) {
       return res.text();
     }).then(function (scriptContent) {
       try {
-        const vmContext = {exports, require, module, global, __filename, __dirname, URL,console,process,Buffer, ...global, remoteEntryName: name};
+        const vmContext = {exports, require, module, global, __filename, __dirname, URL, URLSearchParams, console, process,Buffer, ...global, remoteEntryName: name};
         const remote = vm.runInNewContext(scriptContent + '\\nmodule.exports', vmContext, {filename: 'node-federation-loader-' + name + '.vm'});
-        global.__remote_scope__[name] = remote[name] || remote;
-        global.__remote_scope__._config[name] = url;
-        callback(global.__remote_scope__[name])
+        const foundContainer = remote[name] || remote
+
+        if(!global.__remote_scope__[name]) {
+          global.__remote_scope__[name] = {
+            get: foundContainer.get,
+            init: function(initScope, initToken) {
+              try {
+                return foundContainer.init(initScope, initToken)
+              } catch (e) {
+                return 1
```

Comment:
please add relevant logs

Reply:
catch requires no logs it just means someone else already initialized it - currently i have to blindly try initializing.
In delegate modules i can remove all manual initializations. 
Its not a true error, it just throws if already initialized and we cannot always reliably see that.